src/TuDa.CIMS.Api/Interfaces/IAssetItemService.cs

```diff
@@ -1,11 +1,12 @@
-﻿using TuDa.CIMS.Shared.Dtos;
+﻿using Refit;
+using TuDa.CIMS.Shared.Dtos;
 using TuDa.CIMS.Shared.Entities;
 
 namespace TuDa.CIMS.Api.Interfaces;
 
 public interface IAssetItemService
 {
-    Task<ErrorOr<IEnumerable<AssetItem>>> GetAllAsync();
+    Task<ErrorOr<IEnumerable<AssetItem>>> GetAllAsync([Query] string? nameOrCas);
```

Comment:
🔧 Attribute does nothing here
```suggestion
    Task<ErrorOr<IEnumerable<AssetItem>>> GetAllAsync(string? nameOrCas);
```